doc/changes/changes_1.2.4.md

```diff
@@ -0,0 +1,38 @@
+# The Spark Exasol Connector 1.2.4, released 2023-??-??
+
+Code name:
```

Comment:
```suggestion
Code name: Dependency Upgrade
```

Reply:
Not defined yet, as mentioned above comment.